app/views/course/assessment/assessments/_form.html.slim

```diff
@@ -8,7 +8,7 @@
   = f.input :start_at
   = f.input :end_at
   = f.input :bonus_end_at
-  = f.input :draft
+  = f.input :draft, disabled: @assessment.questions.empty?
```

Comment:
I don't think disabling is a good UI. The user will not know why he cannot un-draft his assessment.